src/is/L42/repl/css/fav/browserconfig.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<browserconfig>
+    <msapplication>
+        <tile>
+            <square150x150logo src="/mstile-150x150.png"/>
```

Comment:
all the urls stuff should either not have the / or have the full (local path, like /css/fab/..)